mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -38,8 +38,12 @@ class wreck_long
 {
 	respawnType = "WRECK";
 	time = 60;
+
```

Comment:
Please could you remove this extra newline for me?

Reply:
No